app/assets/stylesheets/application.scss

```diff
@@ -13,4 +13,21 @@
  *= require_tree .
  *= require_self
  */
- @import "bootstrap";
+//Importing bootstrap
```

Comment:
Consider moving the import statement for bootstrap to the top of the file for better organization and readability.